theme/sass/components/_stats-bar.scss

```diff
@@ -0,0 +1,42 @@
+.stats-bar {
```

Comment:
this code is not part of the hero image.

Reply:
done